tools/delete_timepoint.php

```diff
@@ -0,0 +1,260 @@
+
+            // Delete from conflicts
+            $DB->delete('conflicts_unresolved', array('CommentId1' => $instrument['CommentID']));
+            $DB->delete('conflicts_unresolved', array('CommentId2' => $instrument['CommentID']));
+            $DB->delete('conflicts_resolved', array('CommentId1' => $instrument['CommentID']));
+            $DB->delete('conflicts_resolved', array('CommentId2' => $instrument['CommentID']));
+        }
+        // Delete from flag
+        echo "\n-- Deleting from flag.\n";
+        $DB->delete('flag', array('SessionID' => $sessionID));
+
+        echo "\n-- Deleting from media.\n";
+        $DB->delete('media', array('session_id' => $sessionID));
+
+        // Delete from session
+        echo "\n-- Deleting from session.\n";
+        $DB->delete('session', array('ID' => $sessionID));
+        // Delete from feedback
+        echo "\n-- Deleting from feedback.\n";
+        $DB->delete('feedback_bvl_thread', array('SessionID' => $sessionID));
+        foreach ($feedbackIDs as $id) {
+            $DB->delete('feedback_bvl_entry', array('FeedbackID' => $id));
+        }
+    } elseif ($printToSQL) {
+        // Delete each instrument instance
+        foreach ($instruments as $instrument) {
+            $name = implode(" -> ", $instrument);
+            $output .= "\n-- Deleting Instrument $name.\n";
+            _printResultsSQL($instrument['Test_name'], array('CommentID' => $instrument['CommentID']), $output, $DB);
+
+            // Delete from conflicts
+            _printResultsSQL('conflicts_unresolved', array('CommentId1' => $instrument['CommentID']), $output, $DB);
+            _printResultsSQL('conflicts_unresolved', array('CommentId2' => $instrument['CommentID']), $output, $DB);
+            _printResultsSQL('conflicts_resolved', array('CommentId1' => $instrument['CommentID']), $output, $DB);
+            _printResultsSQL('conflicts_resolved', array('CommentId2' => $instrument['CommentID']), $output, $DB);
+        }
+        // Delete from flag
+        $output .= "\n-- Deleting from flag.\n";
+        _printResultsSQL('flag', array('SessionID' => $sessionID), $output, $DB);
+
+        $output .= "\n-- Deleting from media.\n";
+        _printResultsSQL('media', array('session_id' => $sessionID), $output, $DB);
+
+        // Delete from session
+        $output .= "\n-- Deleting from session.\n";
+        _printResultsSQL('session', array('ID' => $sessionID), $output, $DB);
+        // Delete from feedback
+        $output .= "\n-- Deleting from feedback.\n";
+        _printResultsSQL('feedback_bvl_thread', array('SessionID' => $sessionID), $output, $DB);
+        foreach ($feedbackIDs as $id) {
+            _printResultsSQL('feedback_bvl_entry', array('FeedbackID' => $id), $output, $DB);
+        }
+
+        //export file
+        $filename = __DIR__ . "/../project/tables_sql/DELETE_session_$sessionID.sql";
```

Comment:
Would be nice to print that somewhere, because every new user will have to dig through the code to find where the file is output and if it actually was successful